### PR TITLE
Service power operations in Service UI

### PR DESCRIPTION
**Pull request by AparnaKarve**
Added the Service Power operations for `Start`, `Stop` and `Suspend` in the Service UI.

The buttons for each of the above operations are displayed per service in the Services list view.

Screenshots
------------
screen shot 2016-10-20 at 2 43 12 pm

screen shot 2016-10-20 at 2 59 41 pm

screen shot 2016-10-21 at 11 29 31 am

Links
------
https://www.pivotaltracker.com/n/projects/1613907/stories/127218477

Depends On 
-------------
https://github.com/ManageIQ/manageiq/pull/11888

### CODE REVIEW

**Comment by chriskacerguis** [comment]
@serenamarie125 does this UX meet your approval?

**Comment by chriskacerguis** [comment]
@AparnaKarve Are there associated tests with this?

**Comment by AparnaKarve** [comment]
@chriskacerguis I have added PhantomJS tests here: https://github.com/ManageIQ/manageiq-ui-service/pull/271/commits/2639e1cb6bb673bc895fbb9cd1ca158b6637fcbd

File: client/app/states/services/list/list.state.js [hunk 1]
```diff
@@ -82,6 +84,33 @@
       sortConfig: serviceSortConfig,
     };
 
+    vm.actionButtons = [
+      {
+        name: __('Start'),
```
Comment (chriskacerguis) [inline client/app/states/services/list/list.state.js]: Shouldn't this be N_('Start')

Comment (AparnaKarve) [inline client/app/states/services/list/list.state.js]: Both `N_('Start')` and `__('Start')` are valid in javascript sources.
Notice that the blueprints js uses the `__('something')` format here : 
https://github.com/ManageIQ/manageiq-ui-service/blob/master/client/app/components/blueprints/blueprints-list.directive.js#L65

Comment (chriskacerguis) [inline client/app/states/services/list/list.state.js]: @himdel thoughts on this?  You pointed this out to another developer.

**Comment by chriskacerguis** [comment]
@serenamarie125 does this UX meet your approval?

File: client/app/states/services/list/list.state.js (hunk 1 above)
Comment (AparnaKarve) [inline client/app/states/services/list/list.state.js]: /cc @mzazrivec

Comment (chriskacerguis) [inline client/app/states/services/list/list.state.js]: Cools.  Just wanted to make sure :)

**Comment by chriskacerguis** [comment]
Thanks for your hard work @AparnaKarve !!!  Good job!

**Comment by AparnaKarve** [comment]
@chriskacerguis Thanks for merging.
As I indicated above it depends on the backend PR ManageIQ/manageiq#11888, which is likely to get merged very soon too.

**Comment by gmcculloug** [comment]
@AparnaKarve It is easy for a merger to miss the `depends on` when it is in the description.  It is ok to have it there but it should also be in the title of the PR.  I would think this PR should have also been marked WIP to avoid merging.

Example PR title: `[WIP](Depends on #10976) LbaaS specs for service...`  (see https://github.com/ManageIQ/manageiq/pull/11061).

**Comment by chessbyte** [comment]
@gmcculloug +1 -- a dependent PR really needs to be a WIP until the PR it depends on is merged

**Comment by AparnaKarve** [comment]
> Example PR title: [WIP](Depends on #10976) LbaaS specs for service... (see ManageIQ/manageiq#11061).

@gmcculloug Agreed, should have done that.
I'm guessing we'll have to do more of this in the future (being explicit about PR dependencies in the title) once the repos are split.

**Comment by chessbyte** [comment]
Euwe Backport details:
```sh
$ git log -1
commit 0b0c0e210189da10863097081d5ea577d1626d38
Author: Chris Kacerguis <chriskacerguis@users.noreply.github.com>
Date:   Tue Oct 25 09:59:31 2016 -0500

    Merge pull request #271 from AparnaKarve/service_power_ops
    
    Service power operations in Service UI
    (cherry picked from commit 903ef4e481883b6197b3a24e3b77a07e479eb3c7)
```

**Pull request merged**